cEP-0019.md

```diff
@@ -0,0 +1,476 @@
+# Meta-review System
+
+| Metadata |                                            |
+| -------- | ------------------------------------------ |
+| cEP      | 0019                                       |
+| Version  | 1.0                                        |
+| Title    | Meta-review System                         |
+| Authors  | Boxuan Li <mailto:liboxuan@connect.hku.hk> |
+| Status   | Proposed                                   |
+| Type     | Process                                    |
+
+## Abstract
+
+This cEP describes the details of the process of implementing the meta-review
+system as a part of the
+[GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5188493739819008).
+
+## Background
+
+People respond to review comments on pull requests by giving feedback. Emojis
+make it much easier to give and receive feedback without a long comment thread.
+Those emojis are called
+[reactions](https://developer.github.com/v4/enum/reactioncontent/) on GitHub
+or [award emoji](https://docs.gitlab.com/ee/user/award_emojis.html) on GitLab.
+They are used by people to publicly express their feelings to review comments,
+which provides feedback to reviewers and all other members. However, they are
+often used in an adhoc way.
+
+GitLab is beyond the scope of this project, as GitHub and GitLab have quite
+different APIs and most core projects of coala are based on GitHub. This
+proposal will only discuss reactions on GitHub, but not award emoji on GitLab.
+
+There are 6 types of reactions on GitHub:
+
+| Emoji           | Markdown    | Unicode |
+| --------------- | ----------- | ------- |
+| THUMBS_UP   👍  | `:+1:`      | U+1F44D |
+| THUMBS_DOWN 👎  | `:-1:`      | U+1F44E |
+| LAUGH       😄  | `:smile:`   | U+1F604 |
+| HOORAY      🎉  | `:tada:`    | U+1F389 |
+| CONFUSED    😕  | `:confused` | U+1F615 |
+| HEART       ❤️  | `:heart:`   | U+FE0F  |
```

Comment:
this glyph renders smaller than the others - put it in the last column, or add a new last column, so that the columns dividers line up correctly in monospaced editors.